templates/collectd.conf.j2

```diff
@@ -69,22 +68,22 @@ LoadPlugin syslog
 #	PrintSeverity false
 #</Plugin>
 
-#<Plugin log_logstash>
+{% if installed_version|version_compare("5.5", ">=") %}#<Plugin log_logstash>
 #	LogLevel info
 #	File "/var/log/collectd.json.log"
 #</Plugin>
-
-<Plugin syslog>
-	LogLevel info
-</Plugin>
+{% endif %}
+#<Plugin syslog>
```

Comment:
You probably need to leave the configuration for syslog plugin.
You should move it just after the `LoadPlugin syslog` line.